backend/environment.py

```diff
@@ -0,0 +1,76 @@
+GridSize = 4
+#Tuple is (shape, color, height)
+Grid = [[(0,0,0) for i in range(GridSize)] for j in range(GridSize)]
+
+shapes = {"cube": 1, "pyramid": 2, "box": 3}
```

Comment:
shapes and colors can probably be sets unless the numbers mean something

Reply:
I was using the dicts to convert the strings to ints and also as a hashmap. A two in one ;)

Reply:
SHAPES = set(["some", "thing"])

Reply:
`Grid = [[("","",0) for i in range(GridSize)] for j in range(GridSize)]` as well